geonode_mapstore_client/client/js/components/CardGrid/CardGrid.jsx

```diff
@@ -134,7 +135,8 @@ const InfiniteScrollCardGrid = ({
     scrollContainer,
     downloading,
     getDetailHref,
-    cardLayoutStyle
+    cardLayoutStyle,
+    cardStyle
```

Comment:
please remove cardStyle if it's not used.
I don't see this applied in other components but please double check